nvme-print.c

```diff
@@ -6261,15 +6261,16 @@ void nvme_show_smart_log(struct nvme_smart_log *smart, unsigned int nsid,
 		smart->percent_used);
 	printf("endurance group critical warning summary: %#x\n",
 		smart->endu_grp_crit_warn_sumry);
-	printf("Data Units Read				: %s %s\n",
+	printf("Data Units Read				: %s %s%s%s\n",
 		uint128_t_to_string_tg(le128_to_cpu(smart->data_units_read),
-		human), human ?
-		int128_to_si_string(smart->data_units_read, 1000 * 512) : "");
-	printf("Data Units Written			: %s %s\n",
+		human), human ? "(" : "", human ?
+		int128_to_si_string(smart->data_units_read, 1000 * 512) : "",
+		human ? ")" : "");
```

Comment:
The same function prints the temperature always in Celsius and Kelvin (in brackets). I think I would just always print both strings.